packages/java/maven-plugin/src/main/java/com/vaadin/fusion/maven/GeneratorProcessor.java

```diff
@@ -18,7 +18,7 @@
     private final MavenProject project;
     private String input;
     private String outputDir = "frontend/generated";
-    private Set<GeneratorConfiguration.Plugin> plugins = new LinkedHashSet<>(3);
+    private Set<GeneratorConfiguration.Plugin> plugins = new LinkedHashSet<>(4);
```

Comment:
Is it the number of default plugins? maybe better to extract this magic number to a constant?